plugins/duck.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: duck
+spec:
+  version: v0.0.1
+  homepage: https://github.com/n3wscott/kubectl-duck
+  shortDescription: List custom resources with ducktype support
+  description: |
+    This plugin allows you to list and get custom resources based on ducktype
```

Comment:
As far as I understand, this is purely a companion tool for Knative, right? If so, this should be mentioned in the description. This helps both knative users to recognize relevance and non-users to filter out.

I'm also not entirely sure, how prevalent the term `ducktype` is in knative. Does it need some explanation?

Reply:
I think it's independent of knative, though the concept of duck types are currently hosted in Knative codebase. That said, for example, a k8s Service can fit to "addressable" duck type, and so could some CRDs. So it's a generic concept, but not defined in k8s core. And it can still be used without Knative installed.

Reply:
Yes, it is not just for Knative. Knative currenly champions the ducktype concepts, but things like triggers in tekton are also addressable and if they label their CRDs correctly this tool will pick it up.